opesci/staggeredgrid.py

```diff
@@ -68,9 +68,13 @@ def __init__(self, dimension, index=None, domain_size=None, grid_size=None,
                  time_step=None, stress_fields=None, velocity_fields=None,
                  omp=True, ivdep=True, simd=False, double=False, io=False,
                  expand=True, eval_const=True, output_vts=False,
-                 converge=False, profiling=False):
+                 converge=False, profiling=False, pluto=False):
+
         self.dimension = dimension
 
+        if(pluto):
+            self.template_keys.append('pluto')
```

Comment:
You should just add it to the list above, since the template engine will always check this variable, not just if it is `True`.